definitions/mocha_v2.4.x/flow_>=v0.22.x/mocha_v2.4.x.js

```diff
@@ -0,0 +1,19 @@
+type TestFunction = (() => void | Promise) | ((done : Function) => void);
```

Comment:
In the next version of Flow we're going to start requiring type parameters on parameterized types like `Promise`. Can you specify `Promise<mixed>` here to pre-empt that?